d2renderers/d2latex/latex_test.go

```diff
@@ -0,0 +1,30 @@
+package d2latex
+
+import (
+	"encoding/xml"
+	"testing"
+)
+
+func TestRender(t *testing.T) {
+	txts := []string{
+		`a + b = c`,
+		`\\frac{1}{2}`,
+	}
+	for _, txt := range txts {
+		svg, err := Render(txt)
+		if err != nil {
+			t.Fatal(err)
+		}
+		var xmlParsed interface{}
+		if err := xml.Unmarshal([]byte(svg), &xmlParsed); err != nil {
+			t.Fatalf("invalid SVG: %v", err)
+		}
+	}
+}
+
+func TestRenderError(t *testing.T) {
+	_, err := Render(`\frac{1}{2}`)
+	if err == nil {
+		t.Fatal(err)
```

Comment:
what will it log if `err` is `nil` here? 
also, you can make it an one liner `if _, err := ..`

Reply:
oh right lol